tools/ansible/tasks/wsu/README.md

```diff
@@ -0,0 +1,38 @@
+# Windows Scale Up (WSU) Playbook
+This playbook is responsible for three things:
+- Preparing a Windows instance for joining an OpenShift cluster.
+- Running the WMCB
+- Ensuring the Windows node has successfully joined the cluster
+
+## Prerequisites for using this playbook
+- Access to an OpenShift cluster
+- A Windows Server 2019 instance within the cluster's subnet, with WinRM configured for ansible: https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html#winrm-setup
+    - You can use the [wni](https://github.com/openshift/windows-machine-config-operator/tree/master/tools/windows-node-installer) tool to create a ready Windows instance on certain cloud providers
+    - If you are using a cloud provider, you may have to add a tag to the Windows instance. It will be of the format `key:kubernetes.io/cluster/<infraID>` with `value: owned`.
+      - You can find the infraID in the Ignition config file metadata `metadata.json`
+- A `hosts` file with the required variables defined. See below for an example:
+```
+[win]
+# Address of a node and its Windows password
+<node_ip> ansible_password=<password>
+
+[win:vars]
+# Windows username, typically 'Administrator'
+ansible_user=<username>
+# Address of the OpenShift cluster e.g. "foo.fah.com". This should not include "https://api-" or a port
```

Comment:
Why this restriction of not providing complete endpoint? isn't it confusing?

Reply:
Personally I dont think its confusing. I think most people would put the `api` endpoint by default, not `api-int`, and this makes things easier.

Reply:
Can we provide a command `oc get routes...` that will help the user?